apps/backoffice-v2/src/initialize-session-recording/initialize-session-recording.ts

```diff
@@ -0,0 +1,15 @@
+import { env } from '@/common/env/env';
+import { init as initSaola } from '@saola.ai/browser';
+
+export const initializeSessionRecording = () => {
+  // @TODO: Uncomment when Saola tests are production ready
+  // if (window.location.host.includes('127.0.0.1') || window.location.host.includes('localhost')) {
+  //   return;
+  // }
```

Comment:
we do use domain name for non production environment like 'dev' + 'sb' 
and i suggest to enable it on sb+dev env in order to record sessions

Reply:
It is enabled in dev already, when we get to sb we'll enable it there too (in Saola's settings)